e2e/app/page/notebook-page.ts

```diff
@@ -48,10 +48,10 @@ export default class NotebookPage extends AuthenticatedPage {
     try {
       await this.findRunButton(120000);
     } catch (err) {
-      console.log(`Reloading "${this.documentTitle}" because cannot find the Run button`);
+      console.warn(`Reloading "${this.documentTitle}" because cannot find the Run button`);
       await this.page.reload({waitUntil: ['networkidle0', 'load']});
     }
-    await this.waitForKernelIdle(180000); // 3 minutes
+    await this.waitForKernelIdle(300000); // 5 minutes
```

Comment:
on rare occasions, notebook kernel takes a long time to become ready/idle. It causes flaky tests.